js/components/graph/Graph.js

```diff
@@ -153,6 +154,7 @@ export class Graph extends React.Component {
           } else if (entry.type_ === "Hybrid") {
             hybridsList.push(entry)
           } else if (entry.type_ === "BoolNode") {
+            entry["status"] = "inactive"
```

Comment:
Overall, I don't want you to modify the existing JSON data that's fetched from the server. Instead, you should create a new object that maps bool node id to its status (which can indeed start as inactive).